test/tools/ossfuzz/solc_opt_target.cpp

```diff
@@ -0,0 +1,8 @@
+#include <test/tools/fuzzer.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
```

Comment:
```suggestion
extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
```

Coding style: `const` to the end, `*` and `&` left-aligned and parameters with `_` prefix.